README.md

```diff
@@ -38,18 +38,27 @@ contains 3 fields:
 |---|---|---|---|---|
 | 1 | `silent`  | `boolean` | if false, logs connection and changes to the console | `true` |
 | 2 | `prettify`  | `boolean` | if true, will filter the result and send it in a specific format | `true`|
-| 3 | `middleware`  | `(DataObjectType) => (string \| Object \| Buffer)` | will only work with `prettify:true`. A function for manipulating the prettified data received from the listener before sending it to the queue | [identity function](https://en.wikipedia.org/wiki/Identity_function) |
+| 3 | `retries`  | `number` | amount of retries to connect to rabbit | `5`| 
+
+
+
 
+### Types:
+| #  | field | type | info | default |
```

Comment:
What is the default?

Reply:
there is not default for Queue-Type, the user **must** define a name for the queue